cleanlab/multiannotator.py

```diff
@@ -0,0 +1,436 @@
+    annotator_agreement = get_annotator_agreement(
+        labels_multiannotator=labels_multiannotator,
+        consensus_labels=consensus_labels,
+    )
+
+    # Compute the label quality scores of the consensus labels
+    quality_of_consensus = get_quality_of_consensus(
+        consensus_labels=consensus_labels,
+        pred_probs=pred_probs,
+        num_annotations=num_annotations,
+        annotator_agreement=annotator_agreement,
+        **kwargs,
+    )
+
+    return (consensus_labels, annotator_agreement, quality_of_consensus)
+
+
+def compute_multiannotator_stats(
+    labels_multiannotator: pd.DataFrame,
+    pred_probs: np.array,
+    # label_quality_scores_multiannotator=None, # is this still needed? it is never called in the functioned
+) -> pd.DataFrame:
+    """Returns overall statistics about each annotator.
+
+    Parameters
+    ----------
+    labels_multiannotator : pd.DataFrame
+        2D pandas DataFrame of multiple given labels for each example with shape (N, M),
+        where N is the number of examples and M is the number of annotators.
+        For more details, labels in the same format expected by the :py:func:`get_label_quality_scores_multiannotator <cleanlab.multiannotator.get_label_quality_scores_multiannotator>`.
+
+    pred_probs : np.array
+        An array of shape ``(N, K)`` of model-predicted probabilities, ``P(label=k|x)``.
+        For details, predicted probabilities in the same format expected by the :py:func:`get_label_quality_scores_multiannotator <cleanlab.multiannotator.get_label_quality_scores_multiannotator>`.
+
+    Returns
+    -------
+    annotator_stats : pd.DataFrame
+        pandas DataFrame in which each row corresponds to one annotator (the row IDs correspond to annotator IDs), with columns:
+        - ``overall_quality``: overall quality of a given annotator's labels
+        - ``num_labeled``: number of examples annotated by a given annotator
+        - ``worst_class``: the class that is most frequently mislabeled by a given annotator
+    """
+
+    # Compute overall quality of each annotator's labels
+    overall_label_health_score_df = labels_multiannotator.apply(
+        lambda s: overall_label_health_score(
+            s[pd.notna(s)].astype("int32"), pred_probs[pd.notna(s)], verbose=False
+        )
+    )
+
+    # Compute the number of labels labeled/annotated by each annotator
+    num_labeled = labels_multiannotator.count()
+
+    # Find the worst labeled class for each annotator
+    worst_class = labels_multiannotator.apply(
+        lambda s: _get_worst_class(s[pd.notna(s)].astype("int32"), pred_probs[pd.notna(s)])
+    )
+
+    # Create multi-annotator stats DataFrame from its columns
+    return pd.DataFrame(
+        {
+            "overall_quality": overall_label_health_score_df,
+            "num_labeled": num_labeled,
+            "worst_class": worst_class,
+        }
+    )
+
+
+def get_label_quality_scores_multiannotator(
```

Comment:
what do you think about shortening to: `get_label_quality_multiannotator()` ?

(since this doesn't just return scores)